kn/knative-dockerfile-deploy/build_deploy/README.md

```diff
@@ -46,63 +46,53 @@ spec:
       value: ""
     - name: deleteExisting
       value: "true"
-  - name: buildah-build
+  - name: build-helloworld
```

Comment:
Can we keep the original name `buildah-build` because `build-helloworld` indicates specific to `helloworld`

Reply:
The `kn-service-create` task below is specific to `helloworld` as well. So we would use a generic name in the first task but then in the other task it would be specific to helloworld anyway. Any thoughts?

Reply:
Alright. The original was a little different. The Pipeline was generic and the PipelineRun made it specific to helloworld. Now it's a bit different. The Pipeline itself is more specific. There's this problem with passing params.IMAGE we discussed on slack. I guess the ARGS could be moved back to PipelineRun but then the args would have to hardcode the image name like this:
```
- name: IMAGE
  value: "quay.io/navidshaikh/helloworld-go"
- name: ARGS
  value:
      - "service"
      - "create"
      - "hello"
      - "--revision-name=hello-v1"
      - "--image=quay.io/navidshaikh/helloworld-go"
      - "--env=TARGET=Tekton"
      - "--service-account=kn-deployer-account"
```
How does that look?

Reply:
I'd say this looks quite OK.

Reply:
I see 
I don't have any objection such :smiley: lets keep this way then

Reply:
@navidshaikh WDYT about the suggestion above? It would keep the `Pipeline` quite generic (not specific to helloworld). Hardcoding the image name on two lines in the single file sounds ok-ish. We can't use `--image=$(params.IMAGE)` there because it won't resolve the variable.

Reply:
Yes, the pipeline should be generic and able to function for different set of inputs.
PipelineRun can be edited for different runs. IMO, its okay to repeat the image name until we've better provision for referencing parameters in PipelineRun itself.
+1 for bringing back the ARGS to PipelineRun.

Reply:
Added a commit that fixes that.